doc/source/io.rst

```diff
@@ -660,12 +659,19 @@ If ``skip_blank_lines=False``, then ``read_csv`` will not ignore blank lines:
    If both ``header`` and ``skiprows`` are specified, ``header`` will be
    relative to the end of ``skiprows``. For example:
 
-   .. ipython:: python
+.. ipython:: python
 
-      data = '# empty\n# second empty line\n# third empty' \
-                'line\nX,Y,Z\n1,2,3\nA,B,C\n1,2.,4.\n5.,NaN,10.0'
-      print(data)
-      pd.read_csv(StringIO(data), comment='#', skiprows=4, header=1)
+   data = '\n'.join(['# empty',
```

Comment:
sorry I didn't see it before, but looks like mostly everywhere on this file, what is being used is:
```
data = ('#empty\n'
        '#second empty line'
...
```
I think it would be nice to keep the same for consistency. Or was this the format that was failing in this case?